config/config.yaml

```diff
@@ -105,12 +105,69 @@ atlite:
 renewable:
   onwind:
     cutout: europe-2019-sarah3-era5
+    resource:
+      smooth: true
```

Comment:
looks like smoothing is turned on for onwind as well. is that intentional?

Reply:
I'm about to turn it off.